ludwig/features/image_feature.py

```diff
@@ -377,10 +381,11 @@ def _get_processed_image(img_store):
                     else:
                         return read_image_and_resize(img_store)
 
-                proc_df[feature[PROC_COLUMN]] = backend.df_engine.map_objects(
+                res = backend.df_engine.map_objects(
                     input_df[feature[COLUMN]],
                     _get_processed_image
                 )
+                proc_df[feature[PROC_COLUMN]] = res if res is not None else default_image
```

Comment:
I think `res` needs to be a series here. Can you move this default logic into `_get_processed_image` above?